contracts/wrapper/NameWrapper.sol

```diff
@@ -209,7 +228,6 @@ contract NameWrapper is
      * @param label Label as a string of the .eth domain to wrap
      * @param wrappedOwner Owner of the name in this contract
      * @param fuses Initial fuses to set
-     * @param expiry When the fuses will expire
      * @param resolver Resolver contract address
      * @return Normalised expiry of when the fuses expire
```

Comment:
Do we still return normalised expiry?

Reply:
We just return the expiry now. I guess it could be removed since it can't really be normalised, but doesn't seem like a bad thing to return since when you wrap, you don't immediately have that information to hand. You'd have to call another function.